akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -54,28 +51,31 @@ private[remote] trait InboundCompressions {
 private[remote] final class InboundCompressionsImpl(
   system:         ActorSystem,
   inboundContext: InboundContext,
-  settings:       ArterySettings.Compression) extends InboundCompressions {
-
-  private val stopped = new AtomicBoolean
+  settings:       ArterySettings.Compression,
+  eventSink:      EventSink                  = IgnoreEventSink) extends InboundCompressions {
 
   // None is used as tombstone value after closed
-  // TOOD would be nice if we can cleanup the tombstones
+  // TODO would be nice if we can cleanup the tombstones
+  // FIXME we should be able to remove the tombstones easily now
```

Comment:
@patriknw so the tombstones were added to avoid signalling a closed connection AFAIR, but now we have some garbage lingering around forever. I think next step here could be simply to schedule a cleanup callback once we put the None in?